packages/cashier/src/Components/usdt-side-note.jsx

```diff
@@ -12,7 +12,7 @@ const USDTSideNote = ({ type }) => {
     return (
         <div className='cashier__usdt-side-note'>
             <Text as='h2' weight='bold' color='prominent'>
-                <Localize i18n_default_text='About Tether' />
+                <Localize i18n_default_text={`About Tether (${type === 'eusdt' ? 'Ethereum' : 'Omni'})`} />
```

Comment:
Hi @maryia-deriv, `Localize` will not be able to translate this change correctly, as it doesn't translate variables. It translates anything inside `i18n_default_text` without processing it, therefore, you need to provide two separate Localize blocks, for each ethereum and omni.

Reply:
Done 👍🏻  )